torchvision/transforms.py

```diff
@@ -252,7 +291,14 @@ def __call__(self, img):
         Returns:
             PIL.Image: Padded image.
         """
-        return ImageOps.expand(img, border=self.padding, fill=self.fill)
+        if self.type == "constant":
+            return ImageOps.expand(img, border=self.padding, fill=self.fill)
+        elif self.type == "symmetric":
```

Comment:
Can't you have a single function `_expand` that accepts the `type` as argument, and passes it directly to the original `np.pad`? I don't see why we need the 3 functions with the same code